py-polars/polars/functions/eager.py

```diff
@@ -58,6 +59,8 @@ def concat(
     parallel
         Only relevant for LazyFrames. This determines if the concatenated
         lazy computations may be executed in parallel.
+    strict
+        If True, reject concatenating DataFrames that are not the same height when how=`horizontal`
```

Comment:
Can I suggest not using a double-negative here? I would recommend something like:

```
    strict
        When how='horizontal', require all DataFrames to be the same height.
```